### DISCUSSION TITLE
Other Open-Source Projects Connections

### DISCUSSION THREAD

**Discussion by aliabudiab** [Yawanseh/guidescript#11] [Ideas]
What are your thoughts on expanding the connections to include other open-source projects in addition to OpenAI?

**Comment by Yawanseh** [comment]
Hey @aliabudiab,

Absolutely, that's an excellent thought! If we leverage [LocalAI](https://github.com/go-skynet/LocalAI), it could help us use OpenAI APIs seamlessly, without any alterations to the existing code. It sounds like a simple addition of support to update the API URL should do the trick. So let's jump right in and bring this brilliant idea to life! 🚀🚀🚀
